packages/zoe/src/instanceRecordStorage.js

```diff
@@ -53,46 +57,49 @@ export const makeInstanceRecordStorage = () => {
         },
       },
     };
+    baggage.set('instanceRecord', nextInstanceRecord);
   };
 
   /** @type {GetInstanceRecord} */
   const getInstanceRecord = () => {
     assertInstantiated();
-    return harden(instanceRecord);
+    return harden(baggage.get('instanceRecord'));
```

Comment:
```suggestion
    return baggage.get('instanceRecord'));
```

If it was in baggage, then it was already passable.

Reply:
done